tasks/enable.yml

```diff
@@ -0,0 +1,104 @@
+---
+- name: Check trust compatibility
+  fail:
+    msg: Fapolicyd does not support trust setting fapolicyd_setup_trust
+  ignore_errors: true
```

Comment:
Why `ignore_errors`?

Reply:
Because I want to run all the checks at the beginning so one can see all the denials and not just one. If one of checks fails "Check failed conditions" will fail and stop the execution and the incompatibility is nicely described.